packages/flutter/test/material/animated_icons_test.dart

```diff
@@ -277,8 +301,26 @@ void main() {
     expect(canvas.invocations, const <RecordedCanvasCall>[
       RecordedRotate(math.pi),
       RecordedTranslate(-48, -48),
+      RecordedScale(0.5, 0.5),
     ]);
   });
+
+  testWidgets('Direction has no effect on position of widget',
+      (WidgetTester tester) async {
```

Comment:
Nit: This should be one line

Reply:
You mean to remove the break line after `...widget',` ?

Reply:
Yup!